econml/inference.py

```diff
@@ -830,7 +845,8 @@ def _expand_outputs(self, n_rows):
         assert shape(self.pred)[0] == shape(self.pred_stderr)[0] == 1
         pred = np.repeat(self.pred, n_rows, axis=0)
         pred_stderr = np.repeat(self.pred_stderr, n_rows, axis=0)
-        return NormalInferenceResults(self.d_t, self.d_y, pred, pred_stderr, self.inf_type, self.fname_transformer)
+        return NormalInferenceResults(self.d_t, self.d_y, pred, pred_stderr, self.inf_type,
+                                      self.fname_transformer)
```

Comment:
Should also propagate the names:
```suggestion
                                      self.fname_transformer, self.feat_name, self.output_name, self.treatment_name)
```

Reply:
The changes are out of date, but will address in a separate commit.